tests/query/query-ctor-002.phpt

```diff
@@ -49,6 +49,9 @@ var_dump(new MongoDB\Driver\Query(
 ===DONE===
 <?php exit(0); ?>
 --EXPECTF--
+Deprecated: MongoDB\Driver\Query::__construct(): The 'maxScan' option is deprecated and will be removed in a future release in %squery-ctor-002.php on line %d
```

Comment:
I believe most of our other tests used `in %s on line %d`. I think we can do the same here and avoid necessary changes if this ever gets renamed.

Reply:
Fixed